src/main/java/pacman/DrawComponents.java

```diff
@@ -17,16 +20,22 @@
     public int xG2 = ghost2.getX();
     public int yG2 = ghost2.getY();
     public ListWalls listWalls;
-
+    public Timer timer;
```

Comment:
Please add a blank line before the method

Reply:
you could change the attributes to private.